src/main/java/Server/Server.java

```diff
@@ -0,0 +1,22 @@
+package Server;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@SpringBootApplication
+@RestController
+@ComponentScan({ "Server", "ServiceLayer" })
+public class Server {
```

Comment:
// The purpose of the class: ......
why its important to the flow??

Reply:
I think it is clear from the name... Too much documentation is bad